### PR TITLE
South Africa - Aitken Fortuin - JavaScript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by SerialCarKey**
Completed first exercises

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Review by ShawenHarker — COMMENTED** [review]
Hi Aitken,
Nice work man, really clean code, easy to read and understand what you are doing, but just a pro tip, when you abbreviate your names, can you make  it a slight but more clear.
Keep up the good work man.

File: exercises/I-floats/exercise.js [hunk 1]
```diff
@@ -1,2 +1,15 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+let Total = numberOfMentors + numberOfStudents;
+let TotalNOS = numberOfStudents / Total * 100;
```
Comment (sinopungula) [inline exercises/I-floats/exercise.js:5]: Hey! Love the use of abbreviations, creative.

**Review by sinopungula — COMMENTED** [review]
Just completed my review. Awesome, clean work. Great job!

File: mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+    return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getTotal(a, b) {
-    total = a ++ b;
-
-    return "The total is total"
+    total = a + b;
```
Comment (he-man-david) [inline mandatory/1-syntax-errors.js:12]: Hi Aitken! This is David, your edu buddy, how are ya? While this code works, keep in mind that this is a global variable. Declaring a global variable like this inside a function is discouraged. You can read this https://stackoverflow.com/questions/10525582/why-are-global-variables-considered-bad-practice, or reach out to me if you have more questions.

File: mandatory/4-tax.js [hunk 3]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(sales) {
+  return sales + (20 / 100) * sales;
```
Comment (he-man-david) [inline mandatory/4-tax.js:9]: Can we do this in simpler way?

**Pull request closed without merging**